src/_scss/pages/covid19/_budgetCategoriesTable.scss

```diff
@@ -0,0 +1,90 @@
+.usda-table {
+    margin-top: 0;
+    td {
+        &:first-child {
+            @include span-columns(6 of 12, table);
+            text-align: left;
+        }
+        @include span-columns(2 of 12, table);
+        text-align: right;
+        word-break: break-word;
+    }
+    .table-header {
+        .table-header__label {
+            @include flex(0 1 auto);
+            text-align: right;
+        }
+
+        .table-header__sort {
+            margin-top: auto;
+            margin-bottom: auto;
+        }
+
+        &:not(:first-child) {
+            .table-header__content {
+                justify-content: flex-end;
+            }
+        }
+    }
+}
+.budget-categories-table-def_code {
```

Comment:
Looks like def code is the modifier here, so BEM class name would be `.budget-categories-table_def-code`